src/lib/ext-activitylog.js

```diff
@@ -114,13 +115,20 @@ class Model {
 
 class View {
   constructor() {
+    this.contentWrapper = document.querySelector('.content-wrapper');
     this.logView = document.querySelector('log-view');
-    this.menuContainer = document.querySelector('.menu-container');
-    this.clearLogBtn = document.querySelector('#clearLogBtn');
-    this.saveLogBtn = document.querySelector('#saveLogBtn');
+    this.optionsBtn = document.querySelector('.options-btn');
+    this.clearLogBtn = document.querySelector('.clear-logs-btn');
+    this.optionsDropdown = document.querySelector('.options-dropdown');
+    this.saveLogBtn = document.querySelector('.save-log-btn');
     this.loadLogFile = document.querySelector('input[name="loadLogFile"]');
     this.notice = document.querySelector('.notice');
-    this.logHeading = document.querySelector('.log-heading');
+    this.filterIdTxt = document.querySelector('.filter-tabid');
+    this.pageType = document.querySelector('.page-type');
+    this.menuWrapper = document.querySelector('.menu-wrapper');
+    this.logCounter = document.querySelector('.logs-counter');
```

Comment:
This member is not used anywhere else, either turn it into a variable, or remove it and use something like `.logs-counter > :last-child` (and `:first-child`) instead of the current two lines.

Reply:
turned it into local variable.